Library/Formula/gpac.rb

```diff
@@ -36,9 +36,11 @@ class Gpac < Formula
   depends_on "openjpeg" => :optional
 
   def install
-    args = ["--disable-wx",
+    # If pulseaudio is linked, the script detects it, but fails to link -lpulse.
```

Comment:
Would be good to add just as a second argument to make the diff nicer. Normally wouldn't care but just given the comment about the bit below.